test/dirichletmultinomial.jl

```diff
@@ -0,0 +1,46 @@
+# Tests for DirichletMultinomial
+
+
+using Distributions
+using Base.Test
+
+srand(123)
+
+α = rand(5)
+d = DirichletMultinomial(10, α)
+
+
+# test parameters
+@test length(d) == 5
+@test ntrials(d) == 10
+@test params(d) == (10, α)
+@test d.α == α
+@test d.α0 == sum(α)
+@test partype(d) == eltype(α)
+
+# test statistics
+@test_approx_eq(mean(d), α * (d.n / d.α0))
+p = d.α / d.α0
+@test_approx_eq(var(d), d.n * (d.n + d.α0) / (1 + d.α0) .* p .* (1.0 - p))
+
+# test Evaluation
+d = DirichletMultinomial(10, 5)
+@test !insupport(d, rand(5))
+@test insupport(d, [2, 2, 2, 2, 2])
+@test insupport(d, 2.0 * ones(5))
+@test !insupport(d, 3.0 * ones(5))
+
+for x in [2 * ones(5), [1, 2, 3, 4, 0], [3.0, 0.0, 3.0, 0.0, 4.0]]
```

Comment:
Should prefix the outer array container with `Any` to avoid the deprecation warnings this causes.